### ISSUE TITLE
Combine single Activity screens and transaction details

### ISSUE THREAD

**Issue by hboon** [AlphaWallet/alpha-wallet-ios#2771]
Tomek:

> and we can even think of reducing this empty blank space in "Activity" under the amount by merging Transaction Details with this screen. I know we designed it to let token issuer place something there, but it turns out to be not confirmed. So I suggest a change.

Before|After
-|-
![image](https://user-images.githubusercontent.com/56189/118606508-15befb00-b7ea-11eb-86fd-3de31248bf81.jpeg)|![image](https://user-images.githubusercontent.com/56189/118606514-1788be80-b7ea-11eb-9f28-40c42b32c88b.jpeg)

@colourfreak can you confirm this is the way to go?

**Comment by colourfreak** [comment]
Yes. But if this is just "Received" or "Sent" type of transaction, we don't need this middle screen as it was with "Swap"

Sent USDT
<img width="774" alt="Screenshot 2021-07-16 at 11 59 30" src="https://user-images.githubusercontent.com/51817359/125931081-9387fb81-f600-478d-8674-3c4bc8ec4bcf.png">


Received USDC (wrong title in the screenshot with USDT, so ignore)
<img width="833" alt="Screenshot 2021-07-16 at 12 00 51" src="https://user-images.githubusercontent.com/51817359/125931120-04a78b10-95cb-448b-8fd4-98651b5dc7a1.png">

Swap
<img width="932" alt="Screenshot 2021-07-16 at 12 02 33" src="https://user-images.githubusercontent.com/51817359/125931239-5a5c6e24-e52f-4982-9e4f-1f3950457348.png">

TDLR
If this is swap transaction, add this middle screen where you group these two (Sent and Receive) in one screen.

Prototype:
https://projects.invisionapp.com/share/KU114ACT6GCT
